### DISCUSSION TITLE
Manual Peak Detection

### DISCUSSION THREAD

**Discussion by jean-slavsky** [OpenChrom/openchrom#543] [Q&A] [answered]
Hello,

I was wondering if there is a way to manually select which peaks to integrate in the current version of OpenChrom? I see the MM (manual) option in the direct UI, but I don't see anyway to add peak markers. I checked the manual but didn't see anything.

I am on a Mac.

Thank you!

**Comment by eselmeister** [comment, marked as answer]
@jean-slavsky In the direct peak detector UI, simply select the MM option and draw a baseline under the peaks in the chromatogram view via CTRL + left mouse button pressed. As soon as the left mouse button is released, the peak is calculated.
